modules/launch.nix

```diff
@@ -164,6 +166,10 @@ let
       "--ro-bind-try ${config.flatpak.infoFile or "/.flatpak-info-not-found"} /.flatpak-info"
     ])} ${pkgs.xdg-dbus-proxy}/bin/xdg-dbus-proxy "$@"
   '';
+
+  passthru = {
+    nixpakExtendModules = config._module.args.extendModules;
```

Comment:
I think it'd be fine to just call it `extendModules` instead of `nixpakExtendModules`

Reply:
done